docker-compose.yml

```diff
@@ -80,33 +82,43 @@ services:
       - celestia-zkevm-net
   
   reth:
-    image: ghcr.io/paradigmxyz/reth:v1.3.11
     container_name: reth
+    image: ghcr.io/rollkit/lumen:latest
     ports:
-      - "9001:9001"   # Metrics
-      - "30303:30303" # P2P
-      - "8545:8545"   # HTTP
-      - "8551:8551"   # Auth RPC (Engine API)
+      - "9001:9001" # metrics
+      - "30303:30303" # eth/66 peering
+      - "8545:8545" # rpc
+      - "8551:8551" # engine
+      - "8546:8546" # ws
     volumes:
       - ./testnet/reth:/home/reth
       - reth:/home/reth/eth-home
     entrypoint: /bin/sh -c
     command:
       - |
-          reth node \
-          --chain /home/reth/eth-genesis.json \
-          --metrics 0.0.0.0:9001 \
-          --authrpc.addr 0.0.0.0 \
-          --authrpc.port 8551 \
-          --http \
-          --http.addr 0.0.0.0 \
-          --http.port 8545 \
-          --http.api "eth,net,web3,txpool" \
-          --authrpc.jwtsecret /home/reth/jwt.hex \
-          --datadir /home/reth/eth-home \
```

Comment:
How come this was removed?

I believe this is required in order to maintain the chain db between restarts. The rollkit restart seems to be fixed here, but when this is missing, the volume data expected to be under `/home/reth/eth-home` is then missing on restart, and all contract deployments etc are lost.

For example, when I restart and attempt to query my hyperlane synthetic ERC20 I get:
```shell
cast code 0xa7578551baE89a96C3365b93493AD2D4EBcbAe97
0x
```